lib/hca/enrollment_eligibility/service.rb

```diff
@@ -28,32 +28,31 @@ class Service < Common::Client::Base
         'groupNumber' => 'insuranceGroupCode'
       }.freeze
 
-      MARITAL_STATUSES = %w[
-        Married
-        Never Married
-        Separated
-        Widowed
-        Divorced
+      MARITAL_STATUSES = [
+        'Married',
+        'Never Married',
```

Comment:
After testing, using `%w[]` with an item that needed a space ("Never Married") wasn't working. I then added a `\` after never, but then lint complained, so I just refactored the whole thing 😆